qiskit/providers/ibmq/credentials/updater.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Helper for updating credentials from API 1 to API 2."""
+
+from .credentials import Credentials
+from .configrc import (read_credentials_from_qiskitrc,
+                       remove_credentials,
+                       store_credentials)
+
+
+QE_URL = 'https://quantumexperience.ng.bluemix.net/api'
+QCONSOLE_URL = 'https://q-console-api.mybluemix.net/api'
+
+QE2_URL = 'https://api.quantum-computing.ibm.com/api'
+QCONSOLE2_URL = 'https://api-qcon.quantum-computing.ibm.com/api'
+
+QE2_AUTH_URL = 'https://auth.quantum-computing.ibm.com/api'
```

Comment:
I tend to think of this module as very transition-specific (and short-lived), so I included them here for being able to cope with existing credentials and sort of a temporary exception. In general, we want to avoid hard-coding URLs (URLs might change, and actually there are more URLs for the different environments, etc), so I'd rather not making part of a more general module to avoid the temptation of reusing them elsewhere.